appvalidator/specs/webapps.py

```diff
@@ -318,7 +317,7 @@ class WebappSpec(Spec):
                     "app://*.gaiamobile.org/manifest.webapp": {
                         "expected_type": types.StringTypes,
                         "not_empty": True,
-                        "values": ["2.2", "3.0"]
+                        "value_matches": r"\d+.\d+(?:-\w+)?"
```

Comment:
Sure, that works for me.  Since I'm not using the group anywhere, I thought of making it explicitly non-capturing, but either way is fine.